src/System.Json/src/System.Json.csproj

```
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimum);netstandard2.0;$(NetMinimum)</TargetFrameworks>
+    <!-- System.Json is a library for compat purposes so we want to keep the same identity as it originally shipped within Silverlight. -->
+    <StrongNameKeyId>MicrosoftShared</StrongNameKeyId>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.7.1</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.8.0</VersionPrefix>
+    <AssemblyVersion>2.0.8.0</AssemblyVersion><!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">2.0.8.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">2.0.9.0</AssemblyVersion>
```

I don't think this one was inbox anywhere (but silverlight and old xamarin frameworks) -- so it should version normally.
```suggestion
    <AssemblyVersion>2.0.8.0</AssemblyVersion>
    <AssemblyVersion Condition="'$(IsPackable)' == 'true'">2.0.9.0</AssemblyVersion>
```

---

Should we preserve the comment at least in a different shape?:

```cs
<!-- NO-INCREMENT: This version is frozen for .NET Standard and .NETCoreApp because the assembly ships inbox. -->
```